LondonBritishLibrary/orient/BLorient80.xml

```diff
@@ -54,47 +124,69 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                        
                         <additions>
-                            
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#1 #112 #113"/>
+                                    <desc>Verses of hymns are written in a very crude hand.</desc>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <locus target="#54r"/>
+                                    <desc><persName ref="PRS10450YosabI"/> is mentioned in prayers of <ref target="#ms_i2.3"/>.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <locus target="#55v"/>
+                                    <desc><persName ref="PRS3692egwalas"/> is mentioned in prayers of <ref target="#ms_i2.3"/>.</desc>
+                                </item>
+                                
+                                <item xml:id="e4">
+                                    <locus target="#101v"/>
+                                    <desc>The name of the scribe was <persName ref="PRS13268AsabaR" role="scribe"/>. The name of the owner was
+                                        <persName ref="PRS13269GabraM" role="owner"/>; his father's and mother's names were <persName ref="PRS13270WaldaE"/> and
+                                        <persName ref="PRS13271WalattaM"/>.</desc>
+                                </item>
+                                
+                                <item xml:id="e5">
+                                    <locus target="#44v"/>
+                                    <desc>Later owners were <persName ref="PRS13272LebsaQ"/> and <persName ref="PRS13273WaldaM"/>, mentioned 
+                                    on
+                                    <locus target="#44v"/> and elsewhere.</desc>
+                                </item>
+                            </list>
                         </additions>
                         
                     </physDesc>
                     
                     <history>
                         <origin>
-                            
-                            <origDate/>
+                            <origDate notBefore="1801" notAfter="1803" evidence="reign"/>
                         </origin>
                         <provenance>The manuscript had very probably belonged to a church in the <placeName ref="LOC3577Gondar"/> area, 
                             from where it was taken by
                             <persName ref="PRS9430Tewodros"/> and brought to <placeName ref="INS0101MadhaneAlam"/>. It was then looted 
                             by the British <persName ref="PRS7484Napier">Napier</persName> expedition in <date>1868</date>.</provenance>
-                        
```

Comment:
If I understand the information in the Preface correctly, this manuscript has been purchased by the British Museum and does not belong to those brought by the expedition.